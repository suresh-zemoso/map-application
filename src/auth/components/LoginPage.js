import React, { useEffect } from 'react';
import { makeStyles } from '@material-ui/core/styles';
import LoginBlock from './LoginBlock';
import signout from '../../utils/signout';
import PropTypes from 'prop-types';



const useStyles = makeStyles({
    root: {
        display: 'flex',
        justifyContent: 'center',
        width: '100%',
        height: '100%',
        background: 'white'
    },
});

const defaultProps = {

}

const propTypes = {


}

const LoginPage = (props) => {

    const classes = useStyles(props);

    useEffect(() => {
        signout()
    }, [])
    return (
        <div className={classes.root}>
            <LoginBlock />
        </div>
    );
};


LoginPage.propTypes = propTypes;
LoginPage.defaultProps = defaultProps;

export default LoginPage;