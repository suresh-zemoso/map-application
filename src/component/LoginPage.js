import React, { useEffect } from 'react';
import { makeStyles } from '@material-ui/core/styles';
import LoginBlock from './LoginBlock';
import signout from '../utility/signout';


const useStyles = makeStyles({
    root: {
        display: 'flex',
        justifyContent: 'center',
        width: '100%',
        height: '100%',
        background: 'white'
    },
});


const LoginPage = (props) => {

    const classes = useStyles(props);

    useEffect(() => {
        signout()
    }, [])
    return (
        <div className={classes.root}>
            <LoginBlock />
        </div>
    );
};


export default LoginPage;