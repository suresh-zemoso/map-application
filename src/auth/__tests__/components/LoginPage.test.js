import React from 'react';
import { mount, shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage';

let wrapped = shallow(<LoginPage />);

describe('LoginPage', () => {

    it('should render the LoginPage Component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });

});      