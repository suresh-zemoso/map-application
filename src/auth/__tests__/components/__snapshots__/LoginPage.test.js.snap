// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage should render the LoginPage Component correctly 1`] = `ShallowWrapper {}`;
